lib/manager/nuget/extract/global-manifest.ts

```diff
@@ -0,0 +1,52 @@
+import * as datasourceNuget from '../../../datasource/nuget';
+import { logger } from '../../../logger';
+import { SkipReason } from '../../../types';
+import type { PackageDependency, PackageFile } from '../../types';
+import type { MsbuildGlobalManifest } from '../types';
+
+export function extractMsbuildGlobalManifest(
+  content: string,
+  packageFile: string
+): PackageFile | null {
+  const deps: PackageDependency[] = [];
+  let manifest: MsbuildGlobalManifest;
+
+  try {
+    manifest = JSON.parse(content);
+  } catch (err) {
+    logger.debug({ fileName: packageFile }, 'Invalid JSON');
+    return null;
+  }
+
+  if (!manifest['msbuild-sdks'] && !manifest.sdk?.version) {
+    logger.debug(
+      { fileName: packageFile },
+      'This global.json is not a Nuget file'
+    );
+    return null;
+  }
+
+  if (manifest.sdk?.version) {
+    deps.push({
+      depType: 'sdk',
+      depName: 'dotnet-sdk',
+      currentValue: manifest.sdk?.version,
+      skipReason: SkipReason.UnsupportedDatasource,
+    });
+  }
+
+  for (const depName of Object.keys(manifest['msbuild-sdks'])) {
+    const sdk = manifest['msbuild-sdks'][depName];
+    const currentValue = sdk[0];
```

Comment:
```suggestion
    const currentValue = manifest['msbuild-sdks'][depName];
```

😏 Otherwise you only get first char of version string.